exercises/practice/kindergarten-garden/test_kindergarten_garden.c

```diff
@@ -0,0 +1,164 @@
+#include "kindergarten_garden.h"
+
+#include "test-framework/unity.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void test_kindergarten_garden(const char *diagram, const char *student,
+                                     const plant_t *expected)
+{
+   plant_t actual[4];
+   plants(diagram, student, actual);
+
+   TEST_ASSERT_EQUAL_INT_ARRAY(expected, actual, 4);
+}
+
+static void test_partial_garden_garden_with_single_student(void)
+{
+   const plant_t expected[] = { RADISHES, CLOVER, GRASS, GRASS };
+   test_kindergarten_garden("RC\nGG", "Alice", expected);
+}
+
+static void test_partial_garden_different_garden_with_single_student(void)
+{
+   const plant_t expected[] = { VIOLETS, CLOVER, RADISHES, CLOVER };
```

Comment:
In the tests for almost all other exercises there is a line TEST_IGNORE(); (except in the first test), with an explaining comment in the second test.
see https://github.com/exercism/c/pull/944#discussion_r1468699220